src/data/get_word_embedding.py

```diff
@@ -0,0 +1,110 @@
+'''
+download the word embedding and filter it based on our corpus
+'''
+
+import urllib.request
+import os
+import shutil
+import zipfile
+import re
+import numpy as np
+
+
+DIR_PATH = os.path.dirname(__file__)
+
+GLOVE_WE_URL = 'http://nlp.stanford.edu/data/wordvecs/glove.42B.300d.zip'
+
+WE_FOLDER = os.path.join(DIR_PATH, 'word_embedding')
+ZIP_FILE = os.path.join(WE_FOLDER, 'glove.42B.300d.zip')
+WE_FILE = os.path.join(WE_FOLDER, 'glove.42B.300d.txt')
+# WE_FILE = os.path.join(WE_FOLDER, 'glove.6B.300d.txt')
+
+CORPUS_FILES = [
+    os.path.join(DIR_PATH, 'cornell_movie_dialogs/formatted_movie_lines.txt'),
+    os.path.join(DIR_PATH, 'south_park/general_train.txt'),
+    os.path.join(DIR_PATH, 'south_park/fine_tune.txt'),
+]
+
+OUTPUT_FILES = [
+    os.path.join(WE_FOLDER, 'filtered.glove.42B.300d.part1.txt'),
+    os.path.join(WE_FOLDER, 'filtered.glove.42B.300d.part2.txt')
```

Comment:
this may looks funny, because even filtered word embedding (45k * 300d) exceeds the Github maximum limit 100mb. so I split it to multiple files

it's ugly, i know, but trust me, u dont want to download the 5GB GloVe and refilter every time...